src/components/pages/Cases/CaseTable.js

```diff
@@ -152,6 +150,16 @@ export default function CaseTable(props) {
     console.log(sorter);
   }
 
+  // This is part of the Tabs component
+  function callback(key) {
```

Comment:
This was on line 484 and you deleted it and added it back here, can you explain?